github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -933,7 +933,6 @@ presubmits:
       preset-dind-enabled: "true"
       preset-docker-mirror-proxy: "true"
     name: pull-kubevirt-verify-go-mod
-    optional: true
     run_if_changed: ^vendor/.*|^staging/.*|go\.mod$|go\.sum$
```

Comment:
oops my bad, since we have this you can't / don't need to set always_run

Reply:
Changed again. Anything else?

Reply:
This line needs to be removed. Otherwise the job is not required. We can only have one of the two: either required, or selective run.

Reply:
@rmohr IIRC there is a bug with prow that it won't show it as required in the list at the PR bottom
but if it fails, it should show you "required" in the table of the comment that it adds with the fails summary

Reply:
it will not be considered in the merge queue as required (tide) and also not on the github page. The meaning of optional and run_if_changes are a little bit fuzzy, agreed, but it is not required in the parts which we care about :)

Reply:
@oshoval here the context: https://github.com/kubevirt/kubevirt/pull/8467